.github/dependabot.yml

```diff
@@ -15,3 +15,7 @@ updates:
       - dependency-name: "k8s.io/apimachinery"
       - dependency-name: "k8s.io/client-go"
       - dependency-name: "k8s.io/kubectl"
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
shouldn't we update directory to cmd/kured, where our Dockerfile resides?

Reply:
https://github.com/weaveworks/kured/pull/466 - for some reason I thought dependabot would traverse through sub-directories to figure out where Dockerfiles need updating... :shrug: